autokeras/hypermodels/wrapper.py

```diff
@@ -200,3 +217,497 @@ class GeneralBlock(block_module.Block):
 
     def build(self, hp, inputs=None):
         raise NotImplementedError
+
+
+class SegmentationBlock(block_module.Block):
+    """Block for image segmentation.
+
+    The image blocks is a block choosing from ResNetBlock, XceptionBlock, ConvBlock,
+    which is controlled by a hyperparameter, 'block_type'.
+
+    # Arguments
+        block_type: String. 'resnet', 'xception', 'vanilla'. The type of Block
+            to use. If unspecified, it will be tuned automatically.
+        normalize: Boolean. Whether to channel-wise normalize the images.
+            If unspecified, it will be tuned automatically.
+        augment: Boolean. Whether to do image augmentation. If unspecified,
+            it will be tuned automatically.
+    """
+
+    def __init__(self,
+                 block_type=None,
+                 normalize=None,
+                 augment=None,
+                 OS=None,
+                 alpha=None,
+                 classes=None,
+                 **kwargs):
+        super().__init__(**kwargs)
+        self.block_type = block_type
+        self.normalize = normalize
+        self.augment = augment
+        self.OS = OS
+        self.alpha = alpha
+        self.classes = classes
+
+    def get_config(self):
+        config = super().get_config()
+        config.update({'block_type': self.block_type,
+                       'normalize': self.normalize,
+                       'augment': self.augment,
+                       'OS': self.OS})
+        return config
+
+    def SepConv_BN(self, x, filters, prefix, stride=1, kernel_size=3, rate=1,
+                   depth_activation=False, epsilon=1e-3):
+        """ SepConv with BN between depthwise & pointwise. Optionally
+            add activation after BN Implements right "same" padding for
+            even kernel sizes.
+            Args:
+                x: input tensor
```

Comment:
Follow the docstring style of other functions. State the type of the argument first. Then describe